jpeg/BUILDING.md

```diff
@@ -0,0 +1,32 @@
+# Instructions to build the static libraries
+
+## Current libraries version
+
+The current static libraries are version 2.1.5.1 (see [official binaries](https://sourceforge.net/projects/libjpeg-turbo/files/2.1.5.1/) and [Github Project Page](https://github.com/libjpeg-turbo/libjpeg-turbo)).
+## Darwin and Linux
+The static libraries are extracted from the brew bottles.  
+To do so, you need to have [homebrew](https://brew.sh) installed.
+
+1. Install , libjpeg-turbo : `brew install jpeg-turbo`
```

Comment:
You don't have to install it to extract the bottles.

Reply:
You are right! I thought it was needed to run brew info. Removing this...